blog/config/nav.yml

```diff
@@ -49,6 +49,7 @@ nav:
           - articles/knative-v0-3-autoscaling-a-love-story.md
           - articles/ko-fast-kubernetes-microservice-development-in-go.md
           - articles/build-deploy-manage-modern-serverless-workloads-using-knative-on-kubernetes.md
+          - articles/performance-test-with-slos.md
```

Comment:
We generally add new content to the top of the list rather than the bottom

Reply:
Thank you. I fixed this now.